src/components/implementation/no_interface/boot_test/boot_deps.h

```diff
@@ -0,0 +1,36 @@
+#include <print.h>
+
+#undef assert
+#define assert(node) do { if (unlikely(!(node))) { debug_print("assert error in @ "); *((int *)0) = 0;} } while(0)
+
+//#include <mem_mgr.h>
+#include <sched.h>
+#include <cos_alloc.h>
+#include <cobj_format.h>
+#include <cos_types.h>
+
+static vaddr_t kmem_heap = BOOT_MEM_KM_BASE;
+static unsigned long n_kern_memsets = 0;
+
+//the booter uses this to keep track of each comp mapped in
+struct comp_cap_info {
+	struct cos_compinfo cos_compinfo;
```

Comment:
my personal preference would be to not use struct name as variable name itself, I feel it makes it less readable.